mcs/class/referencesource/mscorlib/system/threading/threadpool.cs

```diff
@@ -1107,8 +1107,8 @@ private bool ValidHandle()
             // This will result in a "leak" of sorts (since the handle will not be cleaned up)
             // but the process is exiting anyway.
             //
-            // During AD-unload, we don�t finalize live objects until all threads have been 
-            // aborted out of the AD.  Since these locked regions are CERs, we won�t abort them 
+            // During AD-unload, we don�t finalize live objects until all threads have been 
+            // aborted out of the AD.  Since these locked regions are CERs, we won�t abort them 
```

Comment:
It already was bad enough, but not it seems even worse! 😄 

(could you fix the unicode here, please?)